### PR TITLE
Admin panel

### PR DESCRIPTION
**Pull request by tkachuktaras**

### CODE REVIEW

File: pages/admin_panel.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<?php
+    require "../blocks/connect_to_db.php";
+    $articlesContent = $mysql->query("SELECT * FROM `articles`");
```
Comment (yurykozhenov) [inline pages/admin_panel.php:3]: Move this line to line 25 (above while)

File: pages/profile.php [hunk 2]
```diff
@@ -36,6 +54,11 @@
                             ?>
                         </div>
                         <div class="change">
+                        <?php 
+                            if($user["admin"] == 1){
```
Comment (yurykozhenov) [inline pages/profile.php:58]: Use if and endif blocks, instead of echoing html

File: pages/delete_article.php [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+<head>
+    <?php require "../blocks/head.php"; ?>
+    <title>Админ панель</title>
+    <link rel="stylesheet" href="/css/index.css">
+    <link rel="stylesheet" href="/css/admin_panel.css">
+    <link rel="stylesheet" href="/css/profile.css?1">
+</head>
+
+<body class="bodyphp">
+<h1>Вы действительно хотите удалить эту статью?</h1>
```
Comment (yurykozhenov) [inline pages/delete_article.php:20]: Add spaces in the beggining

File: pages/create_article.php [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+                <div class="article__title">
+                    <input name="title" id="title" class="input_profile" placeholder="Заголовок">
+                </div>
+                <div class="article__maintext">
+                    <textarea style="resize:none; height:130px;" name="text" id="text" class="input_profile" placeholder="Короткий текст про статью"></textarea>
+                </div>
+            </div>
+        </article>
+        <div class="full_text">
+            <textarea style="resize:mone; width:100%; height:71vh;" name="full_text" id="full_text" class="input_profile" placeholder="Полный текст про статью"></textarea>
```
Comment (yurykozhenov) [inline pages/create_article.php:34]: Move style to css

File: pages/create_article.php [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+<body class="bodyphp">
+    <form class="form_create_article" action = "/pages/confirm_create_article.php" method="post">
+        <article class="article">
+            <img src="" class="article__img" alt="Article Image">
+            <div class="article__block">
+                <div class="article__title">
+                    <input name="title" id="title" class="input_profile" placeholder="Заголовок">
+                </div>
+                <div class="article__maintext">
+                    <textarea style="resize:none; height:130px;" name="text" id="text" class="input_profile" placeholder="Короткий текст про статью"></textarea>
```
Comment (yurykozhenov) [inline pages/create_article.php:29]: Move style to css

File: pages/create_article.php [hunk 6]
```diff
@@ -0,0 +1,42 @@
+<?php 
+    require "../blocks/connect_to_db.php";
+    $articlesContent = $mysql->query("SELECT * FROM `articles`");
```
Comment (yurykozhenov) [inline pages/create_article.php:3]: This code is unused, remove it

File: pages/delete_article.php [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+    <title>Админ панель</title>
+    <link rel="stylesheet" href="/css/index.css">
+    <link rel="stylesheet" href="/css/admin_panel.css">
+    <link rel="stylesheet" href="/css/profile.css?1">
+</head>
+
+<body class="bodyphp">
+<h1>Вы действительно хотите удалить эту статью?</h1> 
+    <div class="btns">
+        <a href="/pages/confirm_delete_article.php?id=<?php echo $row["id"]?>" class="header__btn_div margin_right">Удалить</a>
```
Comment (yurykozhenov) [inline pages/delete_article.php:22]: You can use $_GET["id"] here. And remove $row fetching in line 3 and 4

**Review by yurykozhenov — CHANGES_REQUESTED** [review]

File: pages/change_article.php [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+                <div class="article__title">
+                    <input name="title" id="title" class="input_profile" value="<?php echo $row['title']?>" placeholder="Заголовок">
+                </div>
+                <div class="article__maintext">
+                    <textarea style="width:700px; height:130px;" name="text" id="text" class="input_profile" placeholder="Короткий текст про статью"><?php echo $row['text']?></textarea>
+                </div>
+            </div>
+        </article>
+        <div class="full_text">
+            <textarea style="width:100%; height:71vh;" name="full_text" id="full_text" class="input_profile" placeholder="Полный текст про статью"><?php echo $row['full_text']?></textarea>
```
Comment (ghost) [inline pages/change_article.php:34]: Move style to css

**Pull request merged**